src/pal/src/configure.cmake

```diff
@@ -366,6 +366,9 @@ check_cxx_source_runs("
 #include <stdlib.h>
 #include <time.h>
 #include <sys/time.h>
+#ifdef (__linux__)
```

Comment:
The `__LINUX__` symbol is defined in our root CMakeLists.txt based on what cmake reports